packages/rest/src/lib/RequestManager.ts

```diff
@@ -509,7 +509,9 @@ export class RequestManager extends AsyncEventEmitter<RestEventsMap> {
 			// Strip out all ids
 			.replaceAll(/\d{17,19}/g, ':id')
 			// Strip out reaction as they fall under the same bucket
-			.replace(/\/reactions\/(.*)/, '/reactions/:reaction');
+			.replace(/\/reactions\/(.*)/, '/reactions/:reaction')
+			// Strip out webhook tokens
+			.replace(/\/webhooks\/:id\/[^/]+(?<slash>\/)?/, '/webhooks/:id/:token$<slash>');
```

Comment:
There’s no need to use named groups imo, you can just use the regular $1

Reply:
ESLint gave me a warning idk

Reply:
Ahh ok then, that’s a new one